rest/RESTClient.ts

```diff
@@ -63,12 +70,17 @@ export default class RESTClient {
         const discordBucket = api.headersIn['x-ratelimit-bucket'] as string | undefined;
         if (api.headersIn['x-ratelimit-global'] === 'true') {
           this.globallyRateLimited = true;
+
           setTimeout(() => {
             this.globallyRateLimited = false;
+
+            while (this.queue.length >= 1) this.queue.shift()?.();
           }, api.json.retry_after * 1e3);
+          // eslint-disable-next-line no-undefined
```

Comment:
This shouldn't be needed, I changed the rule a while ago

Reply:
![image](https://user-images.githubusercontent.com/70419415/105794698-ebf37700-5f61-11eb-86e1-c2c7e8b2df4d.png)
Apparently not.

Reply:
Seems fine on my end - you might not have updated your fork